sys-fs/loop-aes/loop-aes-3.7w.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-mod
+
+MY_P="${PN/aes/AES}-v${PV}"
+
+DESCRIPTION="Linux kernel module to encrypt disk partitions with AES cipher"
+HOMEPAGE="http://loop-aes.sourceforge.net/loop-AES.README"
+SRC_URI="http://loop-aes.sourceforge.net/loop-AES/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~sparc ~x86"
+IUSE="cpu_flags_x86_aes extra-ciphers keyscrub cpu_flags_x86_padlock"
+
+DEPEND="app-crypt/loop-aes-losetup"
+
+PATCHES=( "${FILESDIR}"/build-initrd_explicit-losetup.patch )
```

Comment:
We like to prepend these patch names with `${P}`, 
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html